spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -132,6 +132,17 @@ def associate_groups_with_app(app, groups)
       details_for_app(app)
     end
 
+    def valid_name_for(input)
+      latinazed = input.to_slug.transliterate.to_s # remove accents
+      latinazed = latinazed.gsub(/[^0-9A-Za-z\d\s]/, '') # remove non-valid characters
+      # it may result in totaly empty strings that only contain chinese symbols.
+      if latinazed != input
```

Comment:
Right, that makes sense. If the user looks into the developer portal manually they'll probably be surprised either way to see that their app is named differently. How about we show a `UI.warning` to the user saying something like `Your app name includes non-ASCII characters, which are not supported by the Apple Developer Portal. To fix this a unique (internal) name "#{app_name}" has been created for you. Your app's real name "#{original_app_name}" will still show up correctly on iTunes Connect and the App Store`. 

We still need the `transliterate` though so that `ÜberApp` will be created as `UberApp` instead of `berApp <md5>` on the portal.